tt_eager/tt_dnn/op_library/eltwise_unary/eltwise_unary_op.hpp

```diff
@@ -199,7 +199,64 @@ operation::ProgramWithCallbacks eltwise_unary_multi_core(
 inline Tensor run_eltwise_unary(
     const Tensor& input_tensor,
     const std::vector<UnaryWithParam>& ops_chain,
-    const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG) {
+    const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
+    std::optional<Tensor> output_tensor = std::nullopt) {
+    TT_FATAL(ops_chain.size() > 0, "At least 1 unary op must be specified");
```

Comment:
I don't think we should be duplicating the code here for `run_eltwise_unary` and `run_eltwise_unary_cq`. We should only have 1 run fn with cq. Can have cq id default to 0 if not provided.

Reply:
Since there is already another `run_eltwise_unary` with cq_id and a different implementation, I have renamed the existing `run_eltwise_unary` to `run_eltwise_unary_with_output_tensor` and added `output_tensor` and `cq_id` support like we talked about. I have set default cq_id to 0 for ops without queue id support.